MAUI/Cartesian-Charts/Errorbar-Chart.md

```diff
@@ -0,0 +1,579 @@
+        XBindingPath = "ID",
+        YBindingPath = "Coal",
+        HorizontalErrorValue = 0.5,
+        VerticalErrorValue = 50,
+        Mode= Horizontal
+    };
+
+    chart.Series.Add(errorBar);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Horizontal Mode Support in Error Bar Series](Chart-types_images/ModeHorizontal.png)
+
+### Vertical
+
+To view vertical error value, you can set the Mode as Vertical, as shown in the below code example.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:ErrorBarSeries ItemsSource="{Binding EnergyProductions}"
+                          XBindingPath="ID"
+                          YBindingPath="Coal"
+                          VerticalErrorValue="50"
+                          HorizontalErrorValue="0.5"
+                          Mode="Vertical">              
+    </chart:ErrorBarSeries>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    ErrorBarSeries errorBar = new ErrorBarSeries()
+    {
+        ItemsSource = new ViewModel().EnergyProductions,
+        XBindingPath = "ID",
+        YBindingPath = "Coal",
+        HorizontalErrorValue = 0.5,
+        VerticalErrorValue = 50,
+        Mode= Vertical
+    };
+
+    chart.Series.Add(errorBar);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Vertical Mode Support in Error Bar Series](Chart-types_images/ModeVertical.png)
+
+
+## Direction 
+
+The [HorizontalDirection]() and [VerticalDirection]() specifies whether to show positive, negative, or both directions of error values.
+
+[ErrorBarDirection]() contains below values:
+
+*`Both` -  It indicates the actual data point value along with specific amount of positive and negative error values.
+
+*`Plus` -  It indicates the actual data point value along with specific amount of positive error value.
+
+*`Minus`-  It indicates the actual data point value along with specific amount of negative error value.
+
+The following code illustrates how to set the [HorizontalDirection]() and the [VerticalDirection]() values to error bar chart.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:ErrorBarSeries ItemsSource="{Binding EnergyProductions}"
+                          XBindingPath="ID"
+                          YBindingPath="Coal"
+                          VerticalErrorValue="50"
+                          HorizontalErrorValue="0.5"
+                          HorizontalDirection="Plus"
+                          VerticalDirectyion="Minus">
+    </chart:ErrorBarSeries>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    ErrorBarSeries errorBar = new ErrorBarSeries()
+    {
+        ItemsSource = new ViewModel().EnergyProductions,
+        XBindingPath = "ID",
+        YBindingPath = "Coal",
+        HorizontalErrorValue = 0.5,
+        VerticalErrorValue = 50,
+        HorizontalDirection="Plus",
+        VerticalDirectyion="Minus"
+    };
+
+    chart.Series.Add(errorBar);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Horizontal Direction as Both in Error Bar Series](Chart-types_images/BasicRenderingErrorBar.png)
```

Comment:
The code snippet and image were different, add a image which describe in the code snippet.

Reply:
I replace the images